portfolio/src/main/webapp/script.js

```diff
@@ -63,3 +63,23 @@ function tryHarderAndGiveDaisy() {
     document.getElementById("flower-image").src = "https://webstockreview.net/images/daisy-clipart-flower-gif-2.gif";
 
 }
+
+
+
+function getHelloFromButton() {
+  console.log('Fetching from servlet');
+  const responsePromise = fetch('/data');
+  responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+  console.log('Handling the response.');
+  const textPromise = response.text();
+  textPromise.then(addCutieToDom);
+}
+
+function addCutieToDom() {
+  console.log("Adding 'hi cutie' to dom: ");
```

Comment:
Remove console.logs from committed code, if these are used for debugging, I encourage you to get familiar with the chrome debugger. It's fantastic.

https://developers.google.com/web/tools/chrome-devtools/javascript